### DISCUSSION TITLE
Welcome to ISCE2 Discussions!

### DISCUSSION THREAD

**Discussion by rtburns-jpl** [isce-framework/isce2#340] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EJFielding** [comment]
Thanks for setting this up @rtburns-jpl

**Comment by hariiirs** [comment]
I am using ISCE and StaMPS for Sentinel-1 time series processing. All steps in isce are over till ake_single_reference_stack_isce  input_file. Then StaMPS was installed (using make and make install), .bash file was edited and assigned as source. Run: mt_prep_isce 0.4 2 3 50 200. Then opened matlab where I am getting load_isce error and  stamps ps_load_initial_isce(data_inc) error. I explored the StaMPS-4.1-beta/matlab, where load_isce.m file is not available then StaMPS-master was downloaded where load_isce.m file is available under StaMPS-master/matlab folder. Then I restarted the same process in terminal window and then in matlab. This time, load_isce error was not coming but ps_load_initial_isce(data_inc) error iis coming. Can anybody help me for this type of time series processing. 

Thanks
Hari Shankar

**Comment by svenborgstrom** [comment]
Good morning everybody,
this is just to know if ISCE is currently supporting (or going to) PAZ (the TSX Spanish twin) data processing.
In order to get the SLC, I fixed a couple of lines in the "unpackFrame_TSX" script and it works, then I moved to "stackStripMap" and I got the "run_files" folder as well, but on "run_01" (for data cropping) I got many error messages.
I don't have here the list of these messages, sorry, but my feeling is that just changing a couple of lines in a script is probably not the right way to properly manage/process a PAZ data workflow.
Any help would be appreciated, thanks.
Sven.

**Reply by EJFielding** [reply]
Hi Sven,
I don't know if anybody at JPL has started working with PAZ data. I have not seen any PAZ data myself. Is the data format significantly different than the TSX data format?
++Eric

**Reply by svenborgstrom** [reply]
Hi Eric,
no, data format is not significantly different than TSX.

Consider that in July there was an AO from DLR (the TDX coordination team) for a joint exploitation of both TSX/TDX and PAZ data, as the three satellites are delivering the same image products and flying in the same orbital plane (reducing this way the revisit time up to 4 days).

In other words, they consider this combination a sort of constellation so I would expect PAZ data processing should be (more or less) the same of TSX processing. 

B.t.w. if you or one of the colleagues reading this forum is interested, there is an ongoing AO for PAZ data request on La Palma, following the eruption in September.
This is where I got PAZ data.

If someone at JPL is going to work on these data please let me know, as I would like to see how ISCE works in this case. I'm now working with ISCE jointly with MintPy for MT analysis, with very nice results: I would like to get the same also for La Palma.

Cheers,
Sven.

**Comment by gabrielaquintana77** [comment]
Hello everyone! I hope you are ok. My name is Gabriela Quintana Sánchez from Caracas Venezuela, I want use ISCE to study crustal deformation applying SBAS. I have used StaMPS applying PSI in COLM Zulia state to study the subsidence in the zone.
